recipe/meta.yaml

```
@@ -23,6 +23,7 @@ requirements:
     - numpy
     - setuptools
     - cython
+    - extension-helpers
```

The `setup_requires` for this project includes `setuptools-scm`, so this is required here:

```suggestion
    - extension-helpers
    - setuptools-scm
```

However, when I add that on macOS the build still fails with 

```
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: option --no-git not recognized
```

(see change to `build/script`)